### ISSUE TITLE
'Read' status of issue per user

### ISSUE THREAD

**Issue by nagolml** [getsentry/sentry#455]
In sentry there needs to be a visual way fo a user to tell if a they have already viewed an issue.

In a team environment where multiple people are resolving the issues it is hard for the current user to see what  they have already 
reviewed (and determined to be someone else's issue) compared to what still needs to be looked at.

A user should be able to mark an issue as unread if they want so that they can know they need to look at it still.

Basically, this would work and function a lot like an email inbox's read/unread items do. Some of us use the bookmark feature for this purpose... but that is less than ideal.
